src/vs/editor/common/editorCommon.ts

```diff
@@ -4164,6 +4167,33 @@ export var EventType = {
 };
 
 /**
+ * Positions on the view
+ */
+export const ViewPosition = {
```

Comment:
@sandy081 How about the enum gets moved to `oneCursor.ts` as it seems to be an implementation detail of the parsing of the `cursorMove` command and is not editor API -- the strings (as constants) are editor API. We can then call it as we wish, I would still be explicit and use `CursorMoveViewPosition` as we ourselves could get confused a couple years from now.